README.md

```diff
@@ -188,10 +193,11 @@ with an error, or an error will be thrown if no callback is specified.
 * `retry_max_delay`: defaults to `null`. By default every time the client tries to connect and fails time before
 reconnection (delay) almost doubles. This delay normally grows infinitely, but setting `retry_max_delay` limits delay
 to maximum value, provided in milliseconds.
-* `connect_timeout` defaults to `false`. By default client will try reconnecting until connected. Setting `connect_timeout`
-limits total time for client to reconnect. Value is provided in milliseconds and is counted once the disconnect occured.
-* `max_attempts` defaults to `null`. By default client will try reconnecting until connected. Setting `max_attempts`
-limits total amount of reconnects.
+* `connect_timeout` defaults to `86400000`. Setting `connect_timeout` limits total time for client to reconnect.
+Value is provided in milliseconds and is counted once the disconnect occured. The last retry is going to happen exactly at the timeout time.
+That way the default is to try reconnecting until 24h passed.
+* `max_attempts` defaults to `0`. By default client will try reconnecting until connected. Setting `max_attempts`
```

Comment:
I thought so too. But this is actually just a stylistic change, since passing 0 earlier behaved just the same, it was just not documented. So I thought it's probably best to change `null` to `0` and document it.

> How would I tell the client not to reconnect?

Have a look at the next line :D (entering 1 does that - it's about the connection attempts in total not the reconnect attempts. While fixing the max_attempts I first switched it to be the number of reconnects and therefor `0` would have been `don't try to reconnect` but this is not represented by the variable name and did not behave like that before. So this seems to be legit).